### PR TITLE
remove reference to GMT

### PR DESCRIPTION
**Pull request by rabernat**
xref: https://discourse.pangeo.io/t/project-pythia-pangeo-infrastructure-working-group-open-meetings/1061/2

### CODE REVIEW

**Comment by rabernat** [comment]
@charlesbluca -- I was expecting the CI to run on this PR. Do you know why it didn't?

**Comment by charlesbluca** [comment]
That's on me - I hadn't noticed the Travis CI was building the Sphinx docs on PRs, and so I only made a GHA workflow for pushes to the branch. I'll add the workflow now, then reopening the PR should do it.

**Comment by charlesbluca** [comment]
Done! Reopening this should trigger the CI now.

File: docs/meeting-notes.rst [hunk 1]
```diff
@@ -14,8 +14,8 @@ the `meeting notes here <https://docs.google.com/document/d/e/2PACX-1vRerhoxG-wO
 
 **Schedule**
 
-* Early Weeks: 4p GMT
-* Late Weeks: 8p GMT
```
Comment (jhamman) [inline docs/meeting-notes.rst:18]: Just throwing this out there. Do we need two times still? We originally were optimizing for Pacific and UK timezones. I would think the 12pm ET would work for both at this point.

Comment (rabernat) [reply docs/meeting-notes.rst:18]: I think this is part of a broader conversation about what is the point of these meetings and who we want to engage.

**Pull request merged**